core/src/main/java/org/bitcoinj/evolution/QuorumRotationState.java

```diff
@@ -80,8 +80,8 @@
     SimplifiedQuorumList quorumListAtHMinus3C;
     SimplifiedQuorumList quorumListAtHMinus4C;
 
-    LinkedHashMap<Sha256Hash, SimplifiedMasternodeList> mnListsCache = new LinkedHashMap<>();
-    LinkedHashMap<Sha256Hash, SimplifiedQuorumList> quorumsCache = new LinkedHashMap<>();
+    LinkedHashMap<Sha256Hash, SimplifiedMasternodeList> mnListsCache;
+    LinkedHashMap<Sha256Hash, SimplifiedQuorumList> quorumsCache;
```

Comment:
It seems that in Java if we have initializers like this in the class body and also in constructors, the assignments in the constructors would get replaced by this, causing bugs and crashes due to an empty `LinkedHashMap`.